vortex-array/src/array/varbinview/mod.rs

```diff
@@ -195,9 +194,11 @@ impl VarBinViewArray {
         ))
     }
 
+    const BUILDER_BLOCK_SIZE: u32 = 16 * 1024;
```

Comment:
Why are we overriding this?

Reply:
can remove for now, not sure if @robert3005 had a reason

Reply:
I thought 8k was very conservative and 16k was a better default. Mostly after our interview questions

Reply:
yeah, arrow has an issue to increase it because 8K is too conservative https://github.com/apache/arrow-rs/issues/6094